cmd/server/main.go

```diff
@@ -133,29 +147,18 @@ func main() {
 	}
 
 	for _, filename := range filesThatShouldExist {
+		if opts.veryInsecure && (filename == opts.tlsCertPath || filename == opts.tlsKeyPath) {
+			continue
+		}
```

Comment:
@defuse, don't we want to allow the `-no-tls-very-insecure` option? Without this code, `opts.veryInsecure` is unused.